analytics-integrations/uxcam/src/main/java/com/segment/analytics/internal/integrations/UXCamIntegration.java

```diff
@@ -0,0 +1,39 @@
+package com.segment.analytics.internal.integrations;
+
+import android.app.Activity;
+import android.os.Bundle;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.ValueMap;
+import com.segment.analytics.internal.AbstractIntegration;
+import com.uxcam.UXCam;
+/**
+ * UXCam allows you to eliminate customer struggle and improve user experience by capturing
+ * and visualizing screen video and user interaction data.
+ *
+ * @see <a href="http://www.uxcam.com/">UXCam</a>
+ * @see <a href="https://dashboard.uxcam.com/docs#/android">UXCam Android SDK</a>
+ */
+public class UXCamIntegration extends AbstractIntegration<UXCam> {
```

Comment:
Don't you want to be collecting user data here? At least track and identify, probably screen as well.